src/js/entities-service/entities/workspace-patients.js

```diff
@@ -6,6 +7,19 @@ const TYPE = 'workspace-patients';
 const _Model = BaseModel.extend({
   type: TYPE,
   urlRoot: '/api/workspace-patients',
+  setNewStatus(newStatus, patient) {
+    const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
+
+    const attrs = { status: newStatus };
+    const opts = { type: 'PUT' };
+
+    const relationships = {
+      'workspace': this.toRelation(currentWorkspace, 'workspaces'),
+      'patient': this.toRelation(patient, 'patients'),
+    };
+
+    this.save(attrs, { relationships }, opts);
+  },
```

Comment:
A lot of this probably needs to be refactored. This is in the "get it to work" state.

Reply:
Let's make this function a `saveAll` and instead of `newStatus` just pass in `{ status: 'foo' }`

Reply:
Rather than needing to pass the patient here, let's modify the initial entity service request to do:
```js
    return new Model({ id: uuid(patientId, workspaceId), _patient: patientId, _workspace: workspaceId });
```
so that we do not need to re-get the workspace or pass the patient.. the data should always match.

Reply:
Good, updated to that 👍